src/OpenCV_Mat.jl

```diff
@@ -3,70 +3,106 @@
 
 cxx"""
 // Grayscale and binary images 
-template <typename T1>
-inline T1 ptr_val(cv::Mat<T1> &img, int row, int col)
+template <typename T>
+inline T ptr_val(cv::Mat &img, int row, int col)
 {
-  return(static_cast<int>(img.at<T1>(row, col)));
+  return(static_cast<int>(img.at<T>(row, col)));
 }
 
-template <typename T2>
-inline void ptr_val(cv::Mat<T2> &img, int row, int col, double val)
+template <typename T>
+inline void ptr_val(cv::Mat &img, int row, int col, double val)
```

Comment:
Yes, that would be great. I'm stuck with the implementation of `Mat_<_Tp>` constructers.